frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.unit.spec.tsx

```diff
@@ -36,82 +37,70 @@ const QUERY = Question.create({
     ["field", PRODUCTS.TITLE, { "source-field": ORDERS.PRODUCT_ID }],
     "asdf",
   ]);
-
-const [RELATIVE_DAY_FILTER, NUMERIC_FILTER, STRING_CONTAINS_FILTER] =
+const [RELATIVE_DAY_FILTER, NUMERIC_FILTER, STRING_CONTAINS_FILTER]: Filter[] =
   QUERY.filters();
 
 const dummyFunction = jest.fn();
 
+const setup = ({
+  filter: filterElem,
+  query = QUERY,
+  onChange = dummyFunction,
+  onChangeFilter = dummyFunction,
+  showFieldPicker = true,
+}: {
+  filter: Array<any>;
```

Comment:
With [my comment](https://github.com/metabase/metabase/pull/32172#discussion_r1255306945) I meant using `new Field` instead of `as Field` - not to relax the typing.

Reply:
I should have said something about that - the constructor for `Filter` comes from here https://github.com/metabase/metabase/blob/6a071da76e733e3914eac3ebe03b67a85c182e26/frontend/src/metabase-lib/queries/structured/MBQLClause.ts#L9-L13

The only reason why `Filter` was working is because it extends `MBQLClause` which extends `Array`

Reply:
Ah wait I see what you mean, I'll do that. But yeah the typing wasn't a deliberate relaxation, it was just copying the type that's already there 😄